lighthouse-core/gather/drivers/extension.js

```diff
@@ -142,18 +141,16 @@ class ExtensionDriver extends Driver {
           return reject(chrome.runtime.lastError);
         }
 
-        this.url = tabs[0].url;
-        resolve(tabs[0].id);
+        resolve(tabs[0]);
```

Comment:
> Seems bad that a call to getCurrentTabURL() could potentially return a different tab than when connect() is finally called

Thinking about it, I don't think it's a risk. (Although I admit it's a bit odd.)
First `getCurrentTabURL` is called via extension popup to get the URL to do the run with. Later, `connect()` gets the activeTab in order to figure out which tab to control.  But this happens v quickly and even if the user manages to switch to a diff active tab within those 50ms, I think that's okay.